db/migrate/20250304074934_backfill_api_key_scope_modes.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+class BackfillApiKeyScopeModes < ActiveRecord::Migration[7.2]
+  def up
+    DB.exec(<<~SQL)
+      UPDATE api_keys
+      SET scope_mode = (
```

Comment:
After backfilling, do we need to set the default value for this column and set it to `null: false`?

Reply:
Needs to be done separately, especially if this is changed to a post-migration.